src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -317,8 +317,9 @@ public override IOperation VisitBinaryOperator(IBinaryOperation operation, int?
                 return VisitBinaryConditionalOperator(operation, sense: true, captureIdForResult, fallToTrueOpt: null, fallToFalseOpt: null);
             }
 
-            // PROTOTYPE(dataflow): deal with stack spilling
-            return base.VisitBinaryOperator(operation, captureIdForResult);
+            _evalStack.Push(Visit(operation.LeftOperand));
+            IOperation rightOperand = Visit(operation.RightOperand);
+            return new BinaryOperatorExpression(operation.OperatorKind, _evalStack.Pop(), rightOperand, operation.IsLifted, operation.IsChecked, operation.IsCompareText, operation.OperatorMethod, null, operation.Syntax, operation.Type, operation.ConstantValue, operation.IsImplicit);
```

Comment:
>null [](start = 196, length = 4)

Consider using named argument, I assume this is SemanticModel.

Reply:
Fixed

---
In reply to: [164224142](https://github.com/dotnet/roslyn/pull/24465#discussion_r164224142) [](ancestors = 164224142)